haproxy/map.jinja

```diff
@@ -6,6 +6,9 @@
       'role': haproxy_role,
       'this': salt['grains.filter_by']({
         'Debian': {
+          'basic_auth_enable': False,
+          'basic_auth_user': 'admin',
+          'basic_auth_password': 'INVALID_USE_MKPASSWD_TO_CREATE',
```

Comment:
mkpasswd is in the 'whois' package on Ubuntu, for some weird reason.

`mkpasswd -m sha-256` is the actual command needed.


On 30 January 2015 at 14:53, Ash Berlin <notifications@github.com> wrote:

> In haproxy/map.jinja
> <https://github.com/ministryofjustice/haproxy-formula/pull/6#discussion_r23846549>
> :
>
> > @@ -6,6 +6,9 @@
> >        'role': haproxy_role,
> >        'this': salt['grains.filter_by']({
> >          'Debian': {
> > +          'basic_auth_enable': False,
> > +          'basic_auth_user': 'admin',
> > +          'basic_auth_password': 'INVALID_USE_MKPASSWD_TO_CREATE',
>
> Where does mkpasswd come from btw?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/ministryofjustice/haproxy-formula/pull/6/files#r23846549>
> .
>